### ISSUE TITLE
Donkey Kong Country 3 - Random wrong palettes for tiles in intro and title screen

### ISSUE THREAD

**Issue by MP2E** [MiSTer-devel/SNES_MiSTer#104]
This one is tricky because it is random. Sometimes the intro and title screen play with no graphical corruption. But about half the time or a bit more, there are random graphical artifacts in the title screen and in the part where the characters are turning the wheel for the music box.

https://gfycat.com/ambitiousgoodnaturedadeliepenguin

In this clip, the music box part plays with no glitches (until after the 2nd reset), but minor glitches can be seen in the title screen.

If glitches do not show up, simply reset the core until it happens. The glitches can also happen from a fresh start of the core, after powering the MISTer off and on

At first I thought this was a ROM file corruption issue, but the checksum matches No-Intro and SmokeMonster's packs. I also tried the exact same rom on an sd2snes with a 1chip-03 SNES and could not replicate the issue after many resets. I also have a real Donkey Kong Country 3 cart that does not show the issue

EDIT: note that the corruption is usually rather minor, a few tiles are corrupted with the rest looking fine
EDIT2: this was tested as of 3ae74e5c3e66d49007ba1235f601675406c43f47

**Comment by MP2E** [comment]
Just looked into this a bit more and noticed that every tile technically has the right graphics from what I can tell, it is simply the palette that is wrong. I have updated the title of this issue to reflect this. Which tiles get miscolored does seem to be random, though.

**Comment by James-F2** [comment]
I've been trying for an hour now, I can't reproduce this glitch.

Can this be a bad synthezation on quartus side?
Maybe sdram related?

Try this build:
[SNES_Cybernator_20190622.zip](https://github.com/MiSTer-devel/SNES_MiSTer/files/3317465/SNES_Cybernator_20190622.zip)

**Comment by MP2E** [comment]
I tried your build and looped through the intro and title screen over 20 times without any issues. Before it would happen very, very often, so I think it is safe to say I made a bad build somehow. Thank you for looking into this!

I recently tested my SDRAM for 40 minutes using the auto test in the memtest utility and it found 0 errors, so I think it is just the build I created... But I only used the latest sources. So maybe I need to change my Quartus version.
